File: src/com/fusenetworks/fuse/commands/Command_forums.java
package com.fusenetworks.fuse.commands;

import org.bukkit.ChatColor;
import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

@CommandPermissions(source = SourceType.BOTH)
@CommandParameters(description = "Gives a link to the forums", usage = "/<command>", aliases = "forum,forums,fourm,fourms")
public class Command_forums extends BaseCommand {
    @Override
    public boolean run(final CommandSender sender, final Player sender_p, final Command cmd, final String commandLabel, final String[] args, final boolean senderIsConsole) 
    {
        String forums = plugin.getConfig().getString("server.forums");
        String dev = plugin.getConfig().getString("server.dev");
        if (!forums.equalsIgnoreCase("none"))
        {
        sender.sendMessage(ChatColor.AQUA + "Forums: " + forums);
        } else {
            sender.sendMessage(ChatColor.RED + "There are no forums setup for this server");
            if (dev.equalsIgnoreCase("true"))
            {
                sender.sendMessage("Debug Information:");
                sender.sendMessage("Configuration Option: " + forums);
            }
        }
        return true;
    }
}

File: src/com/fusenetworks/fuse/commands/Command_forums.kt
package com.fusenetworks.fuse.commands

import org.bukkit.Bukkit
import org.bukkit.ChatColor
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

abstract class Command_forums : CommandExecutor {
    fun run(sender: CommandSender, sender_p: Player, cmd: Command, commandLabel: String, args: Array<String>, senderIsConsole: Boolean): Boolean {
        val forums = Bukkit.getPluginManager().getPlugin("Fuse").config.getString("server.forums")
        if (!forums.equals("none", ignoreCase = true)) {
            sender.sendMessage(ChatColor.AQUA.toString() + "Forums: " + forums)
        } else {
            sender.sendMessage(ChatColor.RED.toString() + "There are no forums setup for this server")
        }
        return true
    }
}
